test/validators.test.js

```diff
@@ -10688,6 +10688,23 @@ describe('Validators', () => {
           '80912345',
         ],
       },
+      {
+        locale: 'fr-GN',
+        valid: [
+          '+224602123456',
+          '0625123456',
+          '+224635987654',
+          '0722123456',
+          '0661234567',
+        ],
+        invalid: [
+          '+22468123456',
+          '0899123456',
+          '+22462212345',
+          '224722123',
+          'This should fail',
+        ],
+      },
```

Comment:
Same with the tests, can you move these to be in between the tests of `fr-GF` and `fr-GP`?